### DISCUSSION TITLE
Software Update Check Failed after 0.11.12

### DISCUSSION THREAD

**Discussion by MFYDev** [pixelfed/pixelfed#5149] [Support]
Hi there,

Today I just upgraded to the latest github dev code which should be 0.11.12 and however after I logged in to the page I keep seeing this issue

<img width="666" alt="image" src="https://github.com/pixelfed/pixelfed/assets/53750381/788267f8-5353-41bf-b059-1ba81702c717">

Even after the parameter to the env is not working

**Comment by dansup** [comment]
Run `php artisan config:cache && php artisan route:cache`

**Comment by MFYDev** [comment]
Hi I have run it for multiple times. I have been using it for a long time. It's not working, that's why I posted here.

<img width="534" alt="image" src="https://github.com/pixelfed/pixelfed/assets/53750381/2a1067d6-07a9-404a-8af3-0c2207a44251">

The warning is still there

**Comment by dansup** [comment]
Did you try running `php artisan cache:clear` ?
